packages/frontend/tailwind.config.cjs

```diff
@@ -12,5 +10,4 @@ module.exports = {
     corePlugins: {
         preflight: false,
     },
-    plugins: [require("@tailwindcss/line-clamp")],
```

Comment:
Are we sure we don't need this plugin anymore?

Reply:
It was included in the core of the upgraded version of Tailwind. I got a warning right after upgrading so I removed it.